### ISSUE TITLE
guide is misleading

### ISSUE THREAD

**Issue by Kokokokoka** [harrykipper/x210#2]
'(this is NOT recommeded at the moment if you have a NVMe drive. NVMe L1 substates might depend from the ME being operational. If that doesn't work your drive might end up undetected).'
This seems to be purely wrong.

**Comment by Kokokokoka** [comment]
Also,
Enable VMX for virtualization    should be on.

**Comment by Kokokokoka** [comment]
Still, after clean compile, it seems that there is no ec.bin

**Comment by harrykipper** [comment]
> Still, after clean compile, it seems that there is something wrong with ec as keyboard is not working and there are tons of wrong ec messages in dmesg

Please, try again rebuilding it with the current code. I am not seeing any error in dmesg,

**Comment by Kokokokoka** [comment]
I'll try, but for now libgfxinit is not working for me, I have the 2880x1920 
screen though.
You have 1920x1200, how is your screen connected?

**Comment by Kokokokoka** [comment]
Here's the old motherboard pic http://www.cnmod.cn/wp-content/uploads/2019/06/004-1024x768.png my display is connected with another port, not the red box one, also I have to use a switch to replace initing screen

**Comment by harrykipper** [comment]
Ok, so you have a motherboard from the previous batch, while mine's from the third. You should build coreboot with blobs (especially the ME) from your stock firmware, as they are likely to be different. Where is your screen connected, if not to the eDP port? Mine is connected to the eDP, the white one in the red box, and I also had to connect the LVDS cable. 
Could you please extract the vbt from your stock bios (it's in /sys/kernel/debug/dri/0/i915_vbt) and post the output of `intel_vbt_decode vbt.bin | grep T3`?

**Comment by Kokokokoka** [comment]
No. I have a third branch, it was the only photo in the net with board (from a brief search).
Mine Is connected to the eDP port, as far as I can see from the old branch pic.
I had only LVDS setup before, when I was using the x201 display.
I don't have my stock bios image anymore, sadly.

**Comment by Kokokokoka** [comment]
cp -r x210_test/src/mainboard/51nb coreboot/src/mainboard/
cp -r x210_test/src/ec/51nb coreboot/src/ec/
to 
cp -r coreboot-x210_test/src/mainboard/51nb coreboot/src/mainboard/
cp -r coreboot-x210_test/src/ec/51nb coreboot/src/ec/
...
as far as i remember path to vbt.bin is worng it is in the root (while in guide mainb/
same for ec.bin

I've managed to build latest coreboot without any issues, next I'll try the libgfxinit.

**Comment by harrykipper** [comment]
...
> as far as i remember path to vbt.bin is worng it is in the root (while in guide mainb/
> same for ec.bin
> 

The .config provided sets the path of vbt.bin to 3rdparty/blobs/mainboard/51nb as the rest of the blobs.

**Comment by Kokokokoka** [comment]
Yes, I was using the internal config from one of your previous builds.
Still, please, fix the:
```
cp -r coreboot-x210_test/src/mainboard/51nb coreboot/src/mainboard/
cp -r coreboot-x210_test/src/ec/51nb coreboot/src/ec/

```
